src/models/cart-model.js

```diff
@@ -52,6 +52,17 @@ const CartModel = BaseModel.extend({
   },
 
   /**
+    * Gets the sum of quantity of each line item
```

Comment:
Absolutely! `Gets the sum quantity of each line item` is confusing to me tho.
The word `lineItemCount` itself confuses me too 😄 . But i understand there's an existing shared understanding.

I'd remove the `of` since that makes sense to two people already 😄